src/main/java/longah/util/TransactionList.java

```diff
@@ -71,4 +71,51 @@ public void clear() {
     public ArrayList<Transaction> getTransactions() {
         return transactions;
     }
+
+    /**
+     * Returns a String printout the list of transactions stored in the system.
+     */
+    public String listTransactions() {
+        int index = 1;
+        String outString = "";
+        for (Transaction transaction : transactions) {
+            outString = outString + String.format("%d.\n%s", index, transaction) + "\n";
+            index ++;
+        }
+        return outString;
+    }
+
+    /**
+     * Returns a String printout of the list of transactions which the member name is involved as the transaction owner
+     * @param memberName String representation of the name of person to search for
+     */
+    public String findPayments(String memberName) {
+        int index = 1;
+        String outString = String.format("%s owns the following list of transactions.", memberName) + "\n";
+        for (Transaction transaction : transactions) {
+            if (transaction.isOwner(memberName)) {
+                outString = outString + String.format("%d.\n%s", index, transaction) + "\n";
+                index ++;
+            }
+        }
+        return outString;
+    }
+
+    /**
+     * Return a String printout the list of transactions which a person is involved as a payee
+     * @param memberName String representation of the name of person to search for
+     */
+    public String findDebts(String memberName) {
```

Comment:
For debate, do you want to implement this together with the previous method? Might serve as a better gauge of all transactions a `Member` object has been involved in.

Reply:
Probably no. It will be more convenient for the user if he only wants to keep track of ongoing debts to ensure all debts are being paid off, or pending transactions to ensure credits are being recieved.

Reply:
The point of the app is that it should simplify all the transactions though, so the list of transactions may not be indicative of who each `Member` needs to pay.